frontend/index.html

```diff
@@ -10,5 +10,27 @@
 <body style="margin: 0;">
 <div id="app"></div>
 <script type="module" src="/src/main.ts"></script>
+<script>
+    //Скрипт для активации Wake Lock - экран телефона не гаснет
+    if ('wakeLock' in navigator) {
+        let wakeLock = null;
+        const requestWakeLock = async () => {
+            try {
+                wakeLock = await navigator.wakeLock.request('screen');
+                wakeLock.addEventListener('release', () => {
+                });
+            } catch (err) {
+                console.error(`${err.name}, ${err.message}`);
```

Comment:
Как-нибудь так будет информативнее:

```suggestion
                console.error("Ошибка при запросе WakeLock", err);
```